network-conversion-server/src/main/resources/application.yml

```diff
@@ -0,0 +1,21 @@
+server:
+  port: 5003
+
+spring:
+  servlet:
+    multipart:
+      max-file-size: 100MB
+      max-request-size: 103MB
+
+network-store-server:
+  preloading-strategy: COLLECTION
+  base-uri: http://localhost:8080/
+
+
+backing-services:
+    case:
+      base-uri: http://localhost:5000
```

Comment:
We should remove this from the application.yml shipped in the jar, so that the default host values (http://case-server/ and http://geo-data-server/ ) defined in IidmConverterService are used

Reply:
Actually, I think this way is better. When someone wants to start the micro-services on local they can do it without changing anything since everything is already linked to the good port instead of having to change every single app.yml to set the localhost:xxxx. plus, since it's in the java code it's more for developers use. For others the base-uri can be overwritten by any external config.